src/webots/gui/WbUpdatedDialog.cpp

```diff
@@ -0,0 +1,67 @@
+// Copyright 1996-2021 Cyberbotics Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "WbUpdatedDialog.hpp"
+
+WbUpdatedDialog::WbUpdatedDialog(QWidget *parent) : QDialog(parent) {
+  setWindowTitle(tr("Welcome to Webots R2022a"));
+
+  QPixmap webotsLogo("images:webots.png");
+  QLabel *mImage = new QLabel(this);
+  mImage->setGeometry(QRect(16, 15, 50, 50));
+  mImage->setPixmap(webotsLogo);
+  mImage->setScaledContents(true);
+
+  QLabel *mLabelThanks = new QLabel(this);
+  mLabelThanks->setGeometry(QRect(75, 30, 330, 30));
+  mLabelThanks->setText(tr("<b>Thank you for updating to Webots R2022a.</b>"));
```

Comment:
Users who are not updating (using Webots for the first time) are also getting this message...
```suggestion
  mLabelThanks->setText(tr("<b>Thank you for using Webots R2022a.</b>"));
```